File: v2/benchmark/tests/integration/hello_world.py
"""
Simple Hello World function in Python
"""

def hello_world(name: str = "World") -> str:
    """
    Simple Hello World function
    
    Args:
        name (str): Optional name parameter, defaults to 'World'
    
    Returns:
        str: Greeting message
    """
    return f"Hello, {name}!"


# Example usage
if __name__ == "__main__":
    print(hello_world())          # Output: Hello, World!
    print(hello_world("Claude"))  # Output: Hello, Claude!

File: v2/benchmark/archive/old-files/hello_world.js
/**
 * Simple Hello World function in JavaScript
 * @param {string} name - Optional name parameter
 * @returns {string} Greeting message
 */
export function helloWorld(name = 'World') {
  return `Hello, ${name}!`;
}

// Example usage
console.log(helloWorld());        // Output: Hello, World!
console.log(helloWorld('Claude')); // Output: Hello, Claude!